cvat-ui/src/components/setup-webhook-pages/create-webhook-page.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import './styles.scss';
+import React from 'react';
+import { Row, Col } from 'antd/lib/grid';
+import { LeftOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import { useHistory } from 'react-router';
+import SetupWebhookContent from './setup-webhook-content';
+
+function CreateWebhookPage(): JSX.Element {
+    const history = useHistory();
+
+    return (
+        <div className='cvat-create-webhook-page'>
+            <Row justify='center' align='middle'>
+                <Col md={20} lg={16} xl={14} xxl={9}>
+                    <Button className='cvat-webhooks-go-back' onClick={() => history.goBack()} type='link' size='large'>
```

Comment:
``goBack()`` does not work always. I can just open ``webhooks/create`` (BTW it works even not being in an organization).